packages/storage-plus/src/indexed_map.rs

```diff
@@ -244,11 +212,34 @@ where
         Box::new(mapped)
     }
 
+    pub fn range_raw<'c>(
+        &self,
+        store: &'c dyn Storage,
+        min: Option<Bound<'a, K>>,
```

Comment:
So this is type-safe, unless they explicitly create a `Bound::ExclusiveRaw` or such, so we still allow a nice backwards-compatible path.

Nice design.

Reply:
Exactly. The raw types are there only for cases when you explicitly want /need to break the "type-safedness".

Thanks. It was fun to do this, even when it involved a lot of side-tracking of the already existing code.